source/core_sntp_client.c

```diff
@@ -256,8 +256,7 @@ static SntpStatus_t validateContext( const SntpContext_t * pContext )
  * transport interface until either:
  * 1. The requested number of bytes @p packetSize have been sent.
  *                    OR
- * 2. Any byte cannot be sent over the network for the
- * #SNTP_SEND_RETRY_TIMEOUT_MS duration.
+ * 2. Any byte cannot be sent over the network for the @p timeoutMs duration.
```

Comment:
It does not necessarily mean that the byte has been sent over the network. Send will just write to the socket buffer, but the actual delivery of the packet is handled by the operating system.

Reply:
That's a platform level detail which is abstracted out from the library.. From the library's perspective, only the transport interface is the way to know if data has been sent out on the network or not.